### PR TITLE
Issue templates and Funding

### PR DESCRIPTION
**Pull request by robertsLando**
https://github.com/orgs/moscajs/teams/maintainers/discussions/7
https://github.com/moscajs/aedes/pull/411

### CODE REVIEW

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+open_collective: aedes
```
Comment (mcollina) [inline .github/FUNDING.yml]: please let's leave this out for the moment, thanks.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[bug]"
+labels: bug
+assignees: mcollina, gnought, robertsLando
```
Comment (mcollina) [inline .github/ISSUE_TEMPLATE/bug_report.md]: do not assign anything to me please.

**Review by mcollina — CHANGES_REQUESTED** [review]
please remove me from assignees

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by gnought** [comment]
@robertsLando would you mind to remove the assignee line?
It is too restrictive, we may have more collaborators later.

**Comment by robertsLando** [comment]
@gnought I think that until we don't get more collaborators this would be fine. If It is too restrective I can remove it, it's the same for me

**Comment by gnought** [comment]
I think it is best to let either of us to pick that up.

**Comment by robertsLando** [comment]
@gnought Ok I will remove us from assign so

**Comment by robertsLando** [comment]
@gnought done

**Pull request merged**